magenta-lib/src/main/scala/magenta/tasks/AWS.scala

```diff
@@ -431,20 +432,30 @@ trait AWS extends Loggable {
 
   def awsRegion(region: Region): AwsRegion = RegionUtils.getRegion(region.name)
 
-  /* A retry condition that logs errors */
-  class LoggingRetryCondition extends SDKDefaultRetryCondition {
-    def exceptionInfo(e: Throwable): String = {
+  /* A retry condition that logs errors and retries on ParseExceptions.
+   * The parse conditions result under heavy load by all accounts and are not in the default retry policy due to
+   * not knowing whether the query is idempotent or not. There is more discussion about this at
+   * https://github.com/aws/aws-sdk-java/issues/892
+   * */
+  class RiffRaffRetryCondition extends SDKDefaultRetryCondition {
+    private def exceptionInfo(e: Throwable): String = {
       s"${e.getClass.getName} ${e.getMessage} Cause: ${Option(e.getCause).map(e => exceptionInfo(e))}"
     }
+    private def isParseException(exception: AmazonClientException): Boolean = {
+      exception.getCause match {
+        case xse:XMLStreamException if xse.getMessage.startsWith("ParseError at") => true
```

Comment:
Bit sad that this is how we have to identify a ParseError.  Looks like non-null `xse.getLocation` might be equivalent - but probably no better than this.